pkg/summary/summarizers.go

```diff
@@ -204,6 +206,22 @@ func initializeCheckErrors() {
 	logrus.Debugf("GVK Error Mapping not provided, using predefined values")
 }
 
+func checkGeneration(obj data.Object, _ []Condition, summary Summary) Summary {
+	if summary.Attributes != nil && summary.Attributes["hasObservedGeneration"] == true {
+		metadataGenerationStr := obj.String("metadata", "generation")
+		observedGenerationStr := obj.String("status", "observedGeneration")
+		generation, errMetadata := strconv.ParseInt(metadataGenerationStr, 10, 64)
+		observed, _ := strconv.ParseInt(observedGenerationStr, 10, 64)
```

Comment:
We want a unparsed observedGeneration to mean in progress, correct?

Reply:
Right. Let me fix this.

Reply:
Fixed. Similar to https://github.com/kubernetes-sigs/cli-utils/blob/master/pkg/kstatus/status/generic.go#L73